elfloader-tool/src/arch-arm/smp_boot.c

```diff
@@ -43,6 +44,9 @@ void non_boot_main(void)
     /* Initialise any platform-specific per-core state */
     non_boot_init();
```

Comment:
Are all functions replaced with the driver `init_non_boot`? If so, `non_boot_init()` can be deleted.